src/components/Details.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+// import { useLocation } from 'react-router-dom';
+let pictures = [];
+
+const Details = (props) => {
+  const { id } = props;
+
+  pictures = useSelector((state) => state.images);
+
+  const filteredPictures = pictures.filter((picture) => picture.id === id);
+  const {
+    author, blurUrl, grayUrl, height, width, picsumUrl,
+  } = filteredPictures[0];
+
+  return (
+    <main>
+      <h2>{author}</h2>
+      <img src={picsumUrl} alt="image1" />
+      <p>
+        Size:
+        {height}
+        /
+        {width}
+      </p>
+      <h3>{author}</h3>
+      <h3>Filter: Blur</h3>
+      <img src={blurUrl} alt="image3" />
+      <h3>Filter: Grayscale</h3>
+      <img src={grayUrl} alt="image2" />
+      <h3>{author}</h3>
+    </main>
+  );
+};
+
+Details.defaultProps = {
```

Comment:
- [ ] It looks like you have not implemented the `Back` button as it is in the project requirements. the button redirects the user to the Home page 👇 
![image](https://user-images.githubusercontent.com/90222110/196745748-62d1d811-8ae9-4569-b0b8-17afaf9e5d95.png)


**NOTE** I have seen that there is a Home button in the nav bar on the details page. If you think that this is the one referred to, then It would be better **if you would be explicit and change the text to  `Back`** so that It looks like the back button